Dockerfile.test

```diff
@@ -1,8 +1,20 @@
-FROM bats/bats
+FROM bash as bats
```

Comment:
Not sure why we don't use `bats/bats` image but sure

Reply:
The bats/bats image is two years old at this point which bugs me. Once https://github.com/bats-core/bats-core/issues/266 is published to Docker Hub we can switch back.